### PR TITLE
Fix crash on sign in when user is already in a room

### PR DESCRIPTION
**Pull request by tirr-c**
cc @skystar-p

### CODE REVIEW

File: backend/api/sign_views.py [hunk 1]
```diff
@@ -155,9 +155,9 @@ def verify_session(request):
                 'username': username,
             }
 
-            room = cache.get('player-room:' + username)
-            if room is not None:
-                response_data['room_id'] = room['room_id']
+            room_id = cache.get('player-room:' + username)
+            if room_id is not None:
+                response_data['room_id'] = room_id
```
Comment (skystar-p) [inline backend/api/sign_views.py]: 와 이거 나 완전 잘못 짰었네

**Comment by skystar-p** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit 7bf1384 has been approved by `skystar-p`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit 7bf138409a83062ef07c0508414f28ebcd7d695e with merge 248210965f3bc50d5aeb7037a940a7821dba72b0...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/311988278?utm_source=github_status&utm_medium=notification)
Approved by: skystar-p
Pushing 248210965f3bc50d5aeb7037a940a7821dba72b0 to master...

**Pull request merged**